lib/plugins/features/movies/controller.js

```diff
@@ -7,3 +7,29 @@ exports.create = async (payload) => {
 
   return new Movie({ id: movie.id }).fetch();
 };
+
+exports.get = async (query = {}) => {
```

Comment:
This is more of a convention thing, but we'll usually call this `findAll`, since we'll usually have a separate `GET /v1/movies/{id}` controller function called `findById` to differentiate the two `GET` requests.